src/jit/gentree.h

```diff
@@ -919,6 +919,10 @@ struct GenTree
 #define GTF_IND_INVARIANT           0x01000000 // GT_IND   -- the target is invariant (a prejit indirection)
 #define GTF_IND_ARR_INDEX           0x00800000 // GT_IND   -- the indirection represents an (SZ) array index
 
+#if defined(FEATURE_HW_INTRINSICS) && defined(_TARGET_XARCH_)
+#define GTF_IND_HW_FLAGS            GTF_UNSIGNED
```

Comment:
Was `unsigned` currently unused for `GT_IND`/`GT_STOREIND`? If so, are we sure it will never be used in the future?

Reply:
Yes, `GTF_UNSIGNED` it should not appear on indir nodes. Those that happen to deal with small unsigned types do so by using the actual unsigned type (`TYP_UBYTE`, `TYP_USHORT`). That said, there's always a small risk when reusing flag values like that due to the possibility of changing the oper of a node.